test/README.rst

```diff
@@ -261,3 +261,34 @@ Cockpit's CI
 WebUI tests when running in CI they use Cockpit's infrastructure.
 For information on the internals of Cockpit's CI see
 `cockpituous documentation <https://github.com/cockpit-project/cockpituous/tree/main/tasks#readme>`_.
+
+Test Naming Conventions
+-----------------------
+
+The following points outline the naming conventions for the test classes in our test suite.
+
+1. **E2E Test Class Names**
+   - End the class names for end-to-end tests with the suffix `_E2E`.
+   - Example:
+     - Test for EFI-based "Use Free Space" scenario: `TestStorageUseFreeSpaceEFI_E2E`
+     - Test for BIOS-based "Use Free Space" scenario: `TestStorageUseFreeSpaceBIOS_E2E`
+
+2. **EFI and BIOS Testing**
+   - Classes related to EFI or BIOS testing should include `EFI` or `BIOS` in their name to
+     clearly indicate the platform type.
+   - Example:
+     - Test for EFI-based installation: `TestStorageReclaimBIOS_E2E`
+     - Test for BIOS-based installation: `TestStorageReclaimEFI_E2E`
+
+3. **Class Names Based on Test File Names**
+   - Test class names should follow the camel case version of the corresponding test file name.
+   - Example:
+     - Test file: `check-storage-use-free-space-e2e`
+     - Test class name: `TestStorageUseFreeSpace_E2E`
+
+4. **Base Test Class Names**
+   - Base test classes, from which EFI and BIOS-specific test cases will derive,
+     should use the naming convention: `BaseTest${NameOfTheFileCamel}`.
+   - Example:
+     - Base class for `check-storage-use-free-space`: `BaseTestStotageUseFreeSpace`
+     - Then EFI/BIOS classes for EFI/BIOS-based tests: `TestStorageUseFreeSpaceEFI` and `TestStorageUseFreeSpaceBIOS`
```

Comment:
The syntax seems to be broken on GitHub. Please take a look on a rich diff to see it.